docs/DeveloperGuide.adoc

```diff
@@ -798,35 +798,99 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |employer |edit an employee’s contact |update their outdated contact information.
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |employer |view an employee’s salary|view their salary.
 
-_{More to be added}_
+|`* * *` |employer |edit an employee’s salary|update their salary.
+
+|`* * *` |employer |add a new employee |store them in the record.
+
+|`* * *` |employer |delete an existing employee |remove employees that are no longer working.
+
+|`* * *` |employer |update the schedule |show employees the available slots.
+
+|`* * *` |employee |view the schedule |choose when I want to work.
+
+|`* * *` |employee |select from the available slots |work during that slot.
+
+|`* * *` |employer |add outlet |update the app with new outlets
+
+|`* *` |employer |sort employees by name |have a clearer view of all employees’ details.
+
+|`* *` |employer |add a new admin account |have multiple managers.
+
+|`* *` |employer |find an employee by name |locate details of employees without having to go through the entire list.
+
+|`* *` |employer |find employees by outlet |locate details of employees according to the outlet they are working at.
+
+|`* *` |employer |view an employee’s working outlet |view their working outlet.
+
+|`* *` |employer |edit an employee’s working outlet |edit their working outlet.
+
+|`* *` |employer |view an employee’s address|view their address information.
+
+|`* *` |employer |edit an employee’s address|update their outdated address.
+
+|`* *` |employee |edit my contact and address |update my particulars.
+
+|`* *` |employee |view my salary so far for the month |plan my expenses.
+
+|`*` |employer |add an employee’s photo|view it at any time.
+
+|`*` |employer |edit an employee’s photo|change an employee's photo to a newer one.
+
+|`*` |employer |view an employee’s photo|recognize the employee.
+
+|`*` |employer |remove an employee’s photo|remove their old photo.
+
+|`*` |employee |add a photo |be easily identified by my employer.
+
+|`*` |employee |have notification for the selected slot |be assured that I am working at that slot.
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `PTMan` and the *Actor* is the `employer`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Add employee
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Employer enters employee details and working outlet(s)
```

Comment:
Maybe you can reword this `Employer enters employee details and working outlets to add employee to the system`